rest/utilities_testing.go

```diff
@@ -2117,13 +2117,19 @@ func (rt *RestTester) GetKeyspaces() []string {
 	return keyspaces
 }
 
-// getSingleKeyspace the name of the keyspace if there is only one test collection on one database.
+// GetSingleKeyspace the name of the keyspace if there is only one test collection on one database.
 func (rt *RestTester) GetSingleKeyspace() string {
 	db := rt.GetDatabase()
-	require.Equal(rt.TB, 1, len(db.CollectionByID), "Database is configured with more collection")
-	for _, collection := range db.CollectionByID {
+	require.Equal(rt.TB, 1, len(db.CollectionByID), "Database must be configured with only one collection to use this function")
+	for id, collection := range db.CollectionByID {
+		if id == base.DefaultCollectionID {
```

Comment:
Is it worth pushing this handling down into getRESTKeyspace so it gets used by GetKeyspaces as well?